Co-evolution Utilities
======================
The Co-evolution Utilities software suite supports both conservation-based and co-evolutionary analyses of multiple sequence alignments (MSAs). Conserved positions are amino acid positions
that retain the same amino acid throughout evolution. Co-evolving positions are pairs of sites that mutate in a coordinated manner throughout evolution.

The software provides support for:

* Differential conservation analysis among the subfamilies of homologous proteins
* Automated co-evolutionary analyses (with 5 algorithms) using an "ensemble-average" treatment of the MSA
* Comparison of co-evolution scores among homologous protein subfamilies
* Mapping MSA column numbers to a given protein’s amino acid positions
* Generation of shuffled MSAs for determining and subtracting the ”noise” from evolutionary data
* Unthresholded analyses of co-evolutionary scores (Jaccard analysis)
* Centrality analysis of unthresholded, weighted networks created from co-evolutionary scores

The suite was used as part of Parente and Swint-Kruse (2013).  Multiple Co-Evolutionary Networks Are Supported by the Common Tertiary Scaffold of the LacI/GalR Proteins.  PLoS ONE.  http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0084398

For detailed instructions, please see README.pdf.
